### PR TITLE
Rename migration package name for 1.22-rc1

### PR DESCRIPTION
**Pull request by wxiaoguang**
Ref: Propose to restart 1.22 release #30501

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: models/migrations/migrations.go [hunk 1]
```diff
@@ -574,18 +573,20 @@ var migrations = []Migration{
 	// v293 -> v294
 	NewMigration("Ensure every project has exactly one default column", v1_22.CheckProjectColumnsConsistency),
 
-	// Gitea 1.22.0 ends at 294
+	// Gitea 1.22.0-rc0 ends at 294
```
Comment (silverwind) [inline models/migrations/migrations.go:576]: Should we name it like that? RC0 never existed as far as I'm aware, thought I'm not sure if something may have been published to docker etc.

Comment (silverwind) [reply models/migrations/migrations.go:576]: At least no docker tag: https://hub.docker.com/r/gitea/gitea/tags

Comment (wxiaoguang) [reply models/migrations/migrations.go:576]: no github release, but rc0 does exist: https://dl.gitea.com/gitea/1.22.0-rc0/ and https://hub.docker.com/r/gitea/gitea/tags?page=&page_size=&ordering=&name=1.22.0-rc0

Comment (silverwind) [reply models/migrations/migrations.go:576]: ok than it exists in some form at least, ok.

**Review by silverwind — APPROVED** [review]

**Pull request merged**